lib/sentry/client.ex

```diff
@@ -160,6 +160,12 @@ defmodule Sentry.Client do
     end
   end
 
+  @spec validate_and_ignore(keyword()) :: :ignored
+  def validate_and_ignore(opts) when is_list(opts) do
```

Comment:
We can also re-use this function above where we call `NimbleOptions.validate!` if we change it to return the options, like suggested in the other comment.